applications/crossbar/priv/couchdb/schemas/callflows.action.json

```diff
@@ -13,11 +13,14 @@
             "type": "object"
         },
         "data": {
+            "default": {},
             "description": "The data/arguments of the callflow module",
+            "extended-validation": true,
```

Comment:
`kzExtendedValidation` :) I don't know though, was more just commenting on following the camelCase convention.